db/migrate/20210914084059_migrate_stripe_connect_enterprise_to_stripe_sca.rb

```diff
@@ -0,0 +1,8 @@
+class MigrateStripeConnectEnterpriseToStripeSca < ActiveRecord::Migration[6.1]
+  def up
+    execute "UPDATE spree_preferences SET key = replace( key, 'stripe_connect', 'stripe_sca') WHERE key like '/spree/gateway/stripe_connect/%'"
+
+    Rails.cache.delete_if {|key, v| key =~ '/spree/gateway/stripe_connect/enterprise_id/' }
```

Comment:
That's a great idea but I think that it only works with Redis. In my local environment I get an error:
```
NoMethodError: undefined method `delete_if' for #<ActiveSupport::Cache::FileStore:0x00005650efbe4d30>
```

But if this is deployed at the same time as the previous PR which introduces the issue, then we don't need to delete the cache.

Also, the above would need to iterate over all cache keys which is slow. Let's not do it unless we really have to.